portfolio/src/main/webapp/about.html

```diff
@@ -70,7 +70,24 @@ <h3 id="pets">Pets</h3>
           he's been staying with my cousin. But he's still a very cute and
           happy kitty! There are some pictures of him on the gallery page.
       </p>
-      <h3>
+      <!-- TODO: Find a way to make comments and navbar common to all pages
+          without actually repeating HTML code, as it is now. -->
+      <div id="comments">
+        <h3>Comments</h3>
+        <form action="/data" method="POST">
+          Name:
+          <br/>
```

Comment:
nit: should be `<br>`  not `<br />`
https://google.github.io/styleguide/htmlcssguide.html#Document_Type